docs/content/guides/recipes/Terraform/howto-private-registry/snippets/env.bicep

```diff
@@ -0,0 +1,59 @@
+//SECRETSTORE
+import radius as radius
+
+resource secretStoreGithub 'Applications.Core/secretStores@2023-10-01-preview' = {
+  name: 'github'
+  properties:{
+    type: 'generic'
+    data: {
+      // Required value, refers to the personal access token or password of the git platform
+      pat: {
+        value: '<my-access-token>'
+      }
+      // Optional value, refers to the username of the git platform
+      username: {
+        value: '<my-username>'
+      }
+    }
+  }
+}
+//SECRETSTORE
+
+//ENV
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+  name: 'prod'
+  properties: {
+    compute: {
+      kind: 'kubernetes'
+      namespace: 'default'
+    }
+    recipeConfig: {
+      terraform: {
+        authentication:{
+          git:{  
+            pat:{
+              // The hostname of your git platform, such as 'dev.azure.com' or 'github.com'
+              'dev.azure.com':{
+                secret: secretStoreGithub.id
+              }
+            }
+          }          
+        }
+      }
+//ENV
```

Comment:
```
    env: {                
        MY_ENV_VAR_1: 'my_value'
    }
```

Reply:
Hey are these just parameters the `context` object will have access to? @AaronCrawfis should the guide also mention all the optional params? We exclude username in the guide since its an optional field as well but its listed in the reference schema. Do we need to make the snippet have everything?